bedrock/legal/templates/legal/emails/fraud-report.txt

```diff
@@ -10,13 +10,13 @@ A new violating website report has been submitted with the following information
 {{ input_product }}
 
 + Specific product
-{{ input_specific_product|safe|strip_tags }}
+{{ input_specific_product|strip_tags|safe }}
```

Comment:
We should not do this: https://docs.djangoproject.com/en/1.7/ref/templates/builtins/#striptags